packages/react/src/renderToStaticMarkup.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { renderToStaticMarkup as _renderToStaticMarkup } from 'react-dom/server';
+
+/** @typedef {import('react').ReactElement} ReactElement */
+
+/* eslint-disable no-console, jsdoc/require-returns-type */
```

Comment:
```suggestion
/* eslint-disable no-console */
```

Should be fixable by doing `@return {string} Markup.` below

Reply:
Fixed